<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleCameraSettingsWidget</class>
 <widget class="QWidget" name="MultipleCameraSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,1">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Authentication</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="loginEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordEdit">
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="loginLabel">
            <property name="text">
             <string>Login:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="nx::client::desktop::ImageControlWidget" name="imageControlWidget" native="true"/>
       </item>
       <item>
        <widget class="QnLicensesProposeWidget" name="licensingWidget" native="true"/>
       </item>
       <item>
        <widget class="QCheckBox" name="enableAudioCheckBox">
         <property name="text">
          <string>Enable Audio</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRecording">
      <attribute name="title">
       <string>Recording</string>
      </attribute>
      <layout class="QVBoxLayout" name="recordingLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="nx::client::desktop::CameraScheduleWidget" name="cameraScheduleWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="expertTab">
      <attribute name="title">
       <string>Expert</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="nx::client::desktop::CameraExpertSettingsWidget" name="expertSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QnAlertBar" name="alertBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnAlertBar</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/alert_bar.h</header>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::CameraExpertSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/resource_properties/camera/expert_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ImageControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/resource_properties/camera/image_control_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnLicensesProposeWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/licensing/licenses_propose_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::CameraScheduleWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/resource_properties/camera/camera_schedule_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>buttonClicked(QAbstractButton*)</slot>
 </slots>
</ui>
